configure.ac

```diff
@@ -29,19 +32,45 @@ AC_ARG_ENABLE(
   [signature-verification],
   [AS_HELP_STRING([--enable-signature-verification], [Enable signature check (disabled by default)])],
   [AC_DEFINE([SIGNATURES], 1, [Enable signature check as default])]
-
+    SIGVERIFICATION=${enableval}
 )
 
 if test "$enable_signature_verification" = "yes" ; then
 	AC_ARG_WITH([swupdcert],
 		[AS_HELP_STRING([--with-swupdcert=FILE], [swupd verification cert])],
-		[AC_DEFINE_UNQUOTED([SWUPDCERT], ["$withval"], [swupd verification cert])],
+		[AC_DEFINE_UNQUOTED([SWUPDCERT], ["$withval"], [swupd verification cert])]
+		  SWUPDCERT=${withval},
 		[AC_DEFINE([SWUPDCERT], ["ClearLinuxRoot.pem"], [swupd verification cert])])
+		  SWUPDCERT="ClearLinuxRoot.pem"
+		  DEFCERT="(Default) "
 fi
 
+AC_ARG_WITH([contenturl],
+	[AS_HELP_STRING([--with-contenturl=URL], [Default content url])],
+	  [AC_DEFINE_UNQUOTED([CONTENTURL], ["$withval"], [Default content url])]
+	  CONTENTURL=${withval},
+	CONTENTURL='!! Warning !! ContentURL not specified!'
+)
+
+AC_ARG_WITH([versionurl],
+	[AS_HELP_STRING([--with-versionurl=URL], [Default version url])],
+	[AC_DEFINE_UNQUOTED([VERSIONURL], ["$withval"], [Default version url])]
+	  VERSIONURL=${withval},
+	VERSIONURL='!! Warning !! VersionURL not specified!'
+)
+
+AC_ARG_WITH([formatid],
+	[AS_HELP_STRING([--with-formatid=NUM], [Default format identifier])],
+	[AC_DEFINE_UNQUOTED([FORMATID], ["$withval"], [Default format identifier])]
+	  FORMATID=${withval},
+	FORMATID='!! Warning !! FormatID not specified!'
+)
```

Comment:
For these three new options, I would prefer keeping with the existing convention for the AC_ARG_ENABLE options, using AS_IF to check values of variables that are already defined with AC_ARG_WITH. An example:

```
AS_IF([test "$with_formatid" = "yes"], [FORMATID="yes"],
        [test "$with_formatid" = "no"], [AC_MSG_ERROR([--without-formatid not supported])],
        [FORMATID='!! Warning !! FormatID not specified!']
)
```

Or, if you want to support --without-formatid, use:

```
AS_IF([test "$with_formatid" = "yes"], [FORMATID="yes"],
        [FORMATID='!! Warning !! FormatID not specified or set to "no"!']
)
```